### ISSUE TITLE
Property dashboard "Click Rate" is including fallback where other views don't?

### ISSUE THREAD

**Issue by JasonBarnabe** [gitcoinco/code_fund_ads#1184]
Property 230.

- On the dashboard graph, click rate for March 25 is 0.45.
- On the earnings tab, CTR for March 25 is 0.825 (0.000 fallback).

Is dashboard combining paid and fallback? Would it be more useful to show the numbers just for paid?

(Note that the summary on the earnings tab shows a number comparable to 0.825.)

**Comment by andrewmcodes** [comment]
Yes the dashboard is combining paid and fallback. Let me talk to @jdorfman about this. Thanks @JasonBarnabe

**Comment by jdorfman** [comment]
Updates: 

* Last week we made some updates to the dashboard graphs. 
* We now have impressions and will add more in the coming weeks.

![Image 2020-04-07 at 8 57 58 AM](https://user-images.githubusercontent.com/398230/78692253-745b3b00-78ae-11ea-9500-b63c81c0ca52.png)

TODO We need to update the label names

cc @hopsoft

**Comment by coderberry** [comment]
@JasonBarnabe @hopsoft This is not yet possible until we start tracking fallback clicks or paid clicks on our daily rollups. We definitely need this added.
